### PR TITLE
Add Win32 binaries to the releases

### PR DESCRIPTION
**Pull request by AutomatedTester**
This change is [Reviewable](https://reviewable.io/reviews/mozilla/geckodriver/138)

### CODE REVIEW

File: ci.sh [hunk 1]
```diff
@@ -12,31 +12,34 @@ rustup_target_add() {
...
+		prefix=arm-linux-gnueabihf
+		;;
+	arm-unknown-linux-gnueabi)
+		prefix=arm-linux-gnueabi
+		;;
+	mipsel-unknown-linux-musl)
+		prefix=mipsel-openwrt-linux
+		;;
+	i686-pc-windows-gnu)
+		prefix=i686-w64-mingw32
```
Comment (AutomatedTester) [inline ci.sh]: I made a total guess at this so corrections welcome

**Comment by andreastt** [comment]
You can have a look at https://github.com/rust-lang/rust/issues/12859 why cross compilation to i686-pc-windows-gnu is problematic.

**Comment by alexcrichton** [comment]
If you're willing to resort to a bit of trickery, I believe that this can actually be made to work locally today! I hope that we'll [soon add support in rustup](https://github.com/rust-lang-nursery/rustup.rs/issues/298) for doing this through a more official manner, but in the meantime you basically just need to [install our custom `rust-mingw` "component"](https://github.com/alexcrichton/port-of-rust/blob/master/i686-pc-windows-gnu/Dockerfile) which we're shipping for i686-pc-windows-gnu. That Dockerfile builds an image which can cross-compile to i686-pc-windows-gnu for both Rust code and C code (using the native MinGW packages which I believe you're also installing here).

If you need any help with that just let me know!

**Comment by timaharrison** [comment]
Could someone please let us know whether and when 32bit versions of geckodriver will be released?  I see that there are techincal difficulties (sorry don't really understand them), but are these preventing aany more 32bit releases or is it being worked on?

The last 32bit release for v0.8.0 does not appear to be compatible with FF 47 or 48.  

At present I'm in the situation where I'm in a corporate environment where we can't get 64bit test PCs, and we can't downgrade Firefox to a version that supports WebDriver (pre marionette).  Since Firefox no longer supports WebDriver this means that we cannot currently run any tests with Firefox, and are having to rely on Chrome and IE.

**Comment by jgraham** [comment]
Yes, I'm going to investigate the solution proposed by @alexcrichton above, and if that doesn't work set up some other Windows build environment.

**Comment by andreastt** [comment]
Superseded by https://github.com/mozilla/geckodriver/pull/213.

**Pull request closed without merging**